config/index.js

```diff
@@ -14,7 +14,7 @@ const redis_password = process.env.REDIS_PASSWORD || '';
 
 // Crash handler
 process.on('uncaughtException', (err) => {
-  logger.error(err.stack);
+  logger.error(`Uncaught exception: ${err.stack}`);
```

Comment:
```suggestion
  logger.error(`Uncaught exception:\n${err.stack}`);
```

Reply:
The string form of `err.stack` looks like:

```
Error: OK!!!
    at EventEmitter.setStatus (/sysroot/home/wjt/src/endlessm/eos-activation-server/node_modules/ioredis/built/Redis.js:608:19)
    at Socket.<anonymous> (/sysroot/home/wjt/src/endlessm/eos-activation-server/node_modules/ioredis/built/redis/event_handler.js:12:14)
    at Object.onceWrapper (node:events:633:28)
    at Socket.emit (node:events:531:35)
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1596:10)
```

So this to me looks good:

```
2024-09-27T12:52:49.127+00:00 - error: Uncaught exception: Error: OK!!!
    at EventEmitter.setStatus (/sysroot/home/wjt/src/endlessm/eos-activation-server/node_modules/ioredis/built/Redis.js:608:19)
    at Socket.<anonymous> (/sysroot/home/wjt/src/endlessm/eos-activation-server/node_modules/ioredis/built/redis/event_handler.js:12:14)
    at Object.onceWrapper (node:events:633:28)
    at Socket.emit (node:events:531:35)
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1596:10)
```